cpu/sensors.py

```diff
@@ -51,7 +51,12 @@ def check_errors(cmd):
     def setUp(self):
         """
         Check pre-requisites before running sensors command
+        Testcase should be executed only on bare-metal environment.
         """
+        cmd = "cat /proc/cpuinfo | grep platform | awk -F':' '{print $NF}'\
+               | awk '{print $NF}'"
+        if not process.system_output(cmd, shell=True) == "PowerNV":
```

Comment:
please use this way` 'PowerNV' in open('/proc/cpuinfo', 'r').read() ` as invoking process.system() kind of performance over head , 

if not 'PowerNV' in open('/proc/cpuinfo', 'r').read()